src/Microsoft.AspNet.OData.Shared/Routing/Conventions/SelectControllerResult.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.AspNet.OData.Routing.Conventions
     /// implementations of SelectControl(). Any design which makes this class public
```

Comment:
We should probably modify this remark tag comment as we're making the change, or remove it and add any additional comments in the PR description.

Reply:
I think remove is the correct action.